tools/ark-cli/src/commands/export/index.ts

```diff
@@ -0,0 +1,136 @@
+import {Command} from 'commander';
+import {execa} from 'execa';
+import * as fs from 'fs/promises';
+import yaml from 'yaml';
+import type {ArkConfig} from '../../lib/config.js';
+import output from '../../lib/output.js';
+
+const ARK_RESOURCE_TYPES = [
+  'secrets',
```

Comment:
i think we should put these in the `config` object w/ defaults then comment in `sample config yaml` how to customise - cause then people can add/remove their own preferences and it keeps it nicely separate